app/views/welcome/recovery_stories.html.haml

```diff
@@ -4,9 +4,11 @@
       Recovery Stories
 
   #recovery-stories-listing
-    .container
-      #recovery-stories-container{data: {stories_count: @recovery_displays.length}}
-        = render @recovery_displays
+    .container.clearfix
+      .recovery-stories-container{data: {stories_count: @slice1.length}}
```

Comment:
nit: I have a slight preference for making these a quoted key value pair (`"stories-count" => ...`), because the transformation of property names confuses me. Doesn't need to be changed, just a feeling I thought I'd burden you with 🙂

Reply:
no prob thanks -- I may have caught a bug in some more testing just now so I'll take another look at these